cirq/ops/common_channels.py

```diff
@@ -31,10 +31,20 @@
 class AsymmetricDepolarizingChannel(gate_features.SingleQubitGate):
     """A channel that depolarizes asymmetrically along different directions."""
 
-    def __init__(self, p_x: float, p_y: float, p_z: float) -> None:
+    def __init__(self, *args, **kwargs) -> None:
```

Comment:
The arguments should be explicit and type annotations should be added. Also, for backwards compatibility we should default num_qubits to 1:

```python
def __init__(self,
             p_x: Optional[float] = None,
             p_y: ...,
             p_z: ...,
             num_qubits: int = 1,
             ps: Optional[Dict[str, float]] = None) -> None: